Framework/Algorithms/src/PolarizationCorrectionFredrikze.cpp

```diff
@@ -22,26 +24,37 @@ using namespace Mantid::API;
 using namespace Mantid::Kernel;
 using namespace Mantid::Geometry;
 
-namespace {
-
-const std::string pNRLabel("PNR");
-
-const std::string pALabel("PA");
-
-const std::string crhoLabel("Rho");
-
-const std::string cppLabel("Pp");
-
-const std::string cAlphaLabel("Alpha");
+namespace Prop {
+static const std::string FLIPPERS{"Flippers"};
+static const std::string pNRLabel("PNR");
+static const std::string pALabel("PA");
+static const std::string EFFICIENCIES{"Efficiencies"};
+static const std::string INPUT_WORKSPACE{"InputWorkspace"};
+static const std::string OUTPUT_WORKSPACE{"OutputWorkspace"};
+static const std::string INPUT_SPIN_STATES("InputSpinStates");
+static const std::string OUTPUT_SPIN_STATES("OutputSpinStates");
+static const std::string crhoLabel("Rho");
```

Comment:
I don't this this and some of the others in here (`cApLabel`etc.) are Properties of the algorithm, and so should probably be in a different namespace.